Test suite failure with typings and TeeSurface

When building pycairo 1.20.1 in [the conda-forge packaging system][pr], we get build failures on a variety of platforms due to the typings test suite having an issue with TeeSurface:

[pr]: https://github.com/conda-forge/pycairo-feedstock/pull/26

```
=================================== FAILURES ===================================
_________________________________ test_typing __________________________________

    def test_typing():
[...]
        cairo_version = tuple(map(int, cairo.cairo_version_string().split(".")))
        if cairo_version >= (1, 16, 0):
>           assert collect_names(cairo) == collect_names(mod)
E           AssertionError: assert {'ANTIALIAS_B...AS_NONE', ...} == {'ANTIALIAS_B...AS_NONE', ...}
E             Extra items in the right set:
E             'TeeSurface'
E             'TeeSurface.add'
E             'TeeSurface.index'
E             'TeeSurface.remove'
E             Use -v to get the full diff
```

I think we'll try hacking around this by adding `TeeSurface` to the list of platform-dependent surfaces like `XlibSurface` that are ignored within this test, but I have no sense as to whether that's actually a good solution.

--- a/tests/test_typing.py
+++ b/tests/test_typing.py
@@ -36,10 +36,4 @@ def test_typing():
                 names.add(key)
         return names
 
-    # We expose all potential API in the typing stubs, so only check
-    # if it exactly matches the Python module with a new enough cairo
-    cairo_version = tuple(map(int, cairo.cairo_version_string().split(".")))
-    if cairo_version >= (1, 16, 0):
-        assert collect_names(cairo) == collect_names(mod)
-    else:
-        assert collect_names(cairo) <= collect_names(mod)
+    assert collect_names(cairo) <= collect_names(mod)

